### ISSUE TITLE
White bar in immersive hide pill mode

### ISSUE THREAD

**Issue by jfitzroy74** [Magisk-Modules-Repo/HideNavBar#79]
Great work on fixing the gcam lag, although I now have an odd looking white bar/space at the bottom of my screen in immersive hidden pill mode. This was never the case with previous versions. Do you think it is something you could look at please? Pixel 6 Pro, Android 12, March update. Thanks in advance

**Comment by DanGLES3** [comment]
> Great work on fixing the gcam lag, although I now have an odd looking white bar/space at the bottom of my screen in immersive hidden pill mode. This was never the case with previous versions. Do you think it is something you could look at please? Pixel 6 Pro, Android 12, March update. Thanks in advance

Mind providing an screenshot? The Gcam lag fix required setting the navbar to be 1.0dp in height instead of 0dp, it was mostly invisible during my testing, fixing this would involve undoing the fix

**Comment by jfitzroy74** [comment]
> > Great work on fixing the gcam lag, although I now have an odd looking white bar/space at the bottom of my screen in immersive hidden pill mode. This was never the case with previous versions. Do you think it is something you could look at please? Pixel 6 Pro, Android 12, March update. Thanks in advance
> 
> Mind providing an screenshot? The Gcam lag fix required setting the navbar to be 1.0dp (lower values didn't seem to fix the issue) in height instead of 0dp, it was mostly invisible during my testing, fixing this would likely involve undoing the lag fix

Yes of course I'll attach an example. Sorry of it seems I'm being very picky. I would be happy to go back to v11 of it's possible for me to do so? The image attached shows a white line space at the bottom of dark backgrounds. WhatsApp is also very noticeable. I love this module and have been using it for a long time. It's pretty much the main reason I root to get rid of the hideous large dead space around the stock nav bar. Thank you in advance
![Screenshot_20220323-183141](https://user-images.githubusercontent.com/96152680/159771921-b554ddb7-e6c2-407f-9f98-8f5be198560b.png)

**Comment by jfitzroy74** [comment]
This is what it looks like in WhatsApp 
![Screenshot_20220323-183859](https://user-images.githubusercontent.com/96152680/159772513-7fdc6ecf-013f-4a12-9431-033b548203de.png)

**Comment by DanGLES3** [comment]
> > > Great work on fixing the gcam lag, although I now have an odd looking white bar/space at the bottom of my screen in immersive hidden pill mode. This was never the case with previous versions. Do you think it is something you could look at please? Pixel 6 Pro, Android 12, March update. Thanks in advance
> > 
> > 
> > Mind providing an screenshot? The Gcam lag fix required setting the navbar to be 1.0dp (lower values didn't seem to fix the issue) in height instead of 0dp, it was mostly invisible during my testing, fixing this would likely involve undoing the lag fix
> 
> Yes of course I'll attach an example. Sorry of it seems I'm being very picky. I would be happy to go back to v11 of it's possible for me to do so? The image attached shows a white line space at the bottom of dark backgrounds. WhatsApp is also very noticeable. I love this module and have been using it for a long time. It's pretty much the main reason I root to get rid of the hideous large dead space around the stock nav bar. Thank you in advance ![Screenshot_20220323-183141](https://user-images.githubusercontent.com/96152680/159771921-b554ddb7-e6c2-407f-9f98-8f5be198560b.png)

It seems it's definitely the Gcam fix, you may roll back to v11 for the time being, i plan on making the Gcam fix optional on v13, unfortunately there is no perfect fix for this other than hope Google fixes whatever wonkyness goes on with Gcam

**Comment by jfitzroy74** [comment]
Would you be kind enough to provide a download link for v11 please? I can't find one anywhere and I no longer have a local copy. Thanks again for your speedy replies. Great module!

**Comment by DanGLES3** [comment]
> Would you be kind enough to provide a download link for v11 please? I can't find one anywhere and I no longer have a local copy. Thanks again for your speedy replies. Great module!

[Immersive-Gestures-(QS)_v11.0_(38).zip](https://github.com/Magisk-Modules-Repo/HideNavBar/files/8335853/Immersive-Gestures-.QS._v11.0_.38.zip)

Here ya go

**Comment by jfitzroy74** [comment]
Thank you very much for all your help and all your great work with this module. I've installed v11 and back to my preferred state. Cheers

**Comment by DanGLES3** [comment]
Just a reminder that version 13 was released which includes total control over the module parameters, so I'll be closing this issue now, have a good day/night

> Thank you very much for all your help and all your great work with this module. I've installed v11 and back to my preferred state. Cheers

**Comment by jfitzroy74** [comment]
Thank you for the reminder. I have installed v13 and everything is perfect. Excellent work!

Regards,

Jason 
 
  On Mon, 4 Apr 2022 at 7:09, ***@***.***> wrote:   


Just a reminder that version 13 was released which includes total control over the module parameters, so I'll be closing this issue now, have a good day/night


Thank you very much for all your help and all your great work with this module. I've installed v11 and back to my preferred state. Cheers
